launcher/ui/widgets/JavaSettingsWidget.cpp

```diff
@@ -137,6 +175,20 @@ void JavaSettingsWidget::initialize()
     m_maxMemSpinBox->setValue(observedMaxMemory);
     m_permGenSpinBox->setValue(observedPermGenMemory);
     updateThresholds();
+
+    auto button = CustomMessageBox::selectable(this, tr("Auto Java Download"),
+                                               tr("%1 can automatically download the correct Java version for each version of Minecraft..\n"
+                                                  "Do you want to enable Java auto-download?\n")
+                                                   .arg(BuildConfig.LAUNCHER_DISPLAYNAME),
+                                               QMessageBox::Warning, QMessageBox::Yes | QMessageBox::No, QMessageBox::Yes)
```

Comment:
I think this shouldn't be a warning. At least on windows (but probably on every platform) this is very in your face, plays a sound when the warning pops up. Just a regular message box should be just fine I think

Reply:
I changed that to question